scripts/shared/deploy.sh

```diff
@@ -136,6 +136,23 @@ function schedule_dummy_pod() {
     deploy_resource "${RESOURCES_DIR}"/dummypod.yaml "$ns"
 }
 
+function deploy_pfilter_cm() {
+    [[ -z "${cluster_cni[$cluster]}" ]] || return 0
+    local ns="submariner-operator"
+    source "${SCRIPTS_DIR}"/lib/deploy_funcs
+
+    echo "Creating the ${ns} namespace..."
+
+    if [[ -z "${NFTABLES}" ]]; then
+        export NFTABLES="\"false\""
+    else
+        export NFTABLES="\"${NFTABLES}\""
+    fi
+    kubectl create namespace "${ns}" || :
+    deploy_resource "${RESOURCES_DIR}"/nftables-cm.yaml "$ns"
```

Comment:
Let’s keep logs next to the event they’re describing. There’s also no need to re-set `NFTABLES` if it’s already set, or to export it for `deploy_resource`:

```suggestion
    echo "Creating the ${ns} namespace..."
    kubectl create namespace "${ns}" || :

    echo Setting up the packet filter configmap...
    if [[ -z "${NFTABLES}" ]]; then
        NFTABLES=false
    fi
    deploy_resource "${RESOURCES_DIR}"/nftables-cm.yaml "$ns"
```